packages/react-theme/src/index.ts

```diff
@@ -1,6 +1,8 @@
 export * from './themes/index';
 export * from './utils/index';
 
+export { tokens } from './global/tokens';
```

Comment:
I propose to move this file directly to root i.e. `packages/react-theme/src/tokens.ts`:

```suggestion
export { tokens } from './tokens';
```

As actually nothing connects it with `/global` folder.

Reply:
I had trouble thinking of where to put this, but I think I also prefer moving this to the root.